echemdb/data/cv/entry.py

```diff
@@ -100,6 +100,38 @@ def __getitem__(self, name):
         """
         return Descriptor(self.package.descriptor)[name]
 
+    def citation(self, render='text'):
+        r"""
+        Returns a formatted reference for the entry's bibliography either as:
+    
+        J. Doe, et al., Journal Name, volume (YEAR) page, "Title"
+
+        Rendering default is plain text 'text', but can be changed to any format 
+        supported by pybtex, such as markdown 'md', 'latex' or 'html'. 
+
+        EXAMPLES::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry.citation(render='text')
+            'O.Alves et al., Physical Chemistry Chemical Physics, 13 (2011) 6010–6021, "Electrochemistry at Ru(0001) in a flowing CO-saturated electrolyte—reactive and inert adlayer phases"'
+
+        """
+        from pybtex.textutils import abbreviate
+        from pybtex.style.template import field, optional, tag, words, join
+        from pybtex.style.formatting.unsrt import Style
+
+        firstname_inititial = abbreviate(self.bibliography.persons['author'][0].first_names[0])
+        lastname = self.bibliography.persons['author'][0].last_names[0]
+        etal = ' et al.' if len(self.bibliography.persons['author']) > 1 else ''
```

Comment:
The leadingspace of the et al. is wrong I think. The `*` block should surround `et al.` and not ` et al.`.